github/issues.go

```diff
@@ -44,6 +44,7 @@ type Issue struct {
 	Repository       *Repository       `json:"repository,omitempty"`
 	Reactions        *Reactions        `json:"reactions,omitempty"`
 	Assignees        []*User           `json:"assignees,omitempty"`
+	ClosedBy         *User             `json:"closed_by,omitempty"`
```

Comment:
There should be a single new method called `GetClosedBy()`, is that what you saw? It should be similar to #575, that one was `GetLocked()`.